docker/ci/ubuntu/Dockerfile

```diff
@@ -19,8 +19,8 @@ RUN python3 -m pip install -r /etc/greptime/requirements.txt
 
 ARG TARGETARCH
 
-ADD $TARGETARCH/$TARGET_BIN /greptime/bin/
+ADD $TARGETARCH/${TARGET_BIN} /greptime/bin/
```

Comment:
Why don't we update `$TARGETARCH` to `${TARGETARCH}`?

May you give some background what this patch fixes and how we can reproduce the "bug"?

Reply:
Thanks for your explanation :D

> I am modifying the dockerfile, not the final commit in this pr.

Then I'll switch this PR to a draft. Feel free to ping me as a reviewer when it is ready for review.

I suggest that we apply a consistent argument usage flavor in the Dockerfile so others can follow and reduce potential bugs.